atlas.hcl

```diff
@@ -1,28 +1,24 @@
 description = "A modern tool for managing database schemas"
-binaries = ["atlas"]
 homepage = "https://atlasgo.io/"
-source = "https://release.ariga.io/atlas/atlas-${os}-${arch_}-v${version}"
-vars = {
-  "arch_": "${arch}",
-}
+binaries = ["atlas"]
+test = "atlas version"
 
-platform "darwin" "arm64" {
```

Comment:
This has to remain, and `${arch_}` must be used, otherwise on M1 Mac's Hermit will try to download `atlas-darwin-arm64-v<version>`, which doesn't exist.

Reply:
https://release.ariga.io/atlas/atlas-linux-arm64-latest

Thankfully `latest` now supports it.